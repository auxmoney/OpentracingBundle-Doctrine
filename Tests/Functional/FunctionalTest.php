<?php

declare(strict_types=1);

namespace Auxmoney\OpentracingDoctrineDBALBundle\Tests\Functional;

use Auxmoney\OpentracingBundle\Tests\Functional\JaegerFunctionalTest;

class FunctionalTest extends JaegerFunctionalTest
{
    public function testStandard(): void
    {
        $this->setUpTestProject('dbal-standard');

        $this->runInTestProject(['symfony', 'console', 'test:doctrine:setup-table']);
        $this->runInTestProject(['symfony', 'console', 'test:doctrine:tracing']);
        self::assertSame(true, true);
    }
}
