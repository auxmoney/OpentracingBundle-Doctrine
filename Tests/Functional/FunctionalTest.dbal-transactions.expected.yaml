operationName: 'test:doctrine:dbal-transactions'
tags:
  -
    key: command.exit-code
    value: 0
children:
  -
    operationName: 'DBAL: TRANSACTION'
    tags:
      -
        key: db.transaction.end
        value: commit
    children:
      -
        operationName: 'DBAL: SELECT test_table'
        tags:
          -
            key: db.statement
            value: 'SELECT COUNT(*) FROM test_table WHERE str IS NOT NULL'
          -
            key: db.parameters
            value: '[]'
      -
        operationName: 'DBAL: INSERT INTO test_table'
        tags:
          -
            key: db.statement
            value: 'INSERT INTO test_table (str) VALUES (?)'
          -
            key: db.parameters
            value: '["a"]'
      -
        operationName: 'DBAL: SELECT test_table'
        tags:
          -
            key: db.statement
            value: 'SELECT COUNT(*) FROM test_table WHERE str IS NOT NULL'
          -
            key: db.parameters
            value: '[]'
      -
        operationName: 'DBAL: SELECT test_table'
        tags:
          -
            key: db.statement
            value: 'SELECT id FROM test_table WHERE str IS NOT NULL'
          -
            key: db.parameters
            value: '[]'
      -
        operationName: 'DBAL: UPDATE test_table'
        tags:
          -
            key: db.statement
            value: 'UPDATE test_table SET str = ? WHERE id = ?'
          -
            key: db.parameters
            value: '[null,"1"]'
      -
        operationName: 'DBAL: SELECT test_table'
        tags:
          -
            key: db.statement
            value: 'SELECT COUNT(*) FROM test_table WHERE str IS NOT NULL'
          -
            key: db.parameters
            value: '[]'
  -
    operationName: 'DBAL: SELECT test_table'
    tags:
      -
        key: db.statement
        value: 'SELECT COUNT(*) FROM test_table WHERE str IS NOT NULL'
      -
        key: db.parameters
        value: '[]'
  -
    operationName: 'DBAL: TRANSACTION'
    tags:
      -
        key: db.transaction.end
        value: rollBack
    children:
      -
        operationName: 'DBAL: INSERT INTO test_table'
        tags:
          -
            key: db.statement
            value: 'INSERT INTO test_table (str) VALUES (?)'
          -
            key: db.parameters
            value: '["a"]'
      -
        operationName: 'DBAL: SELECT test_table'
        tags:
          -
            key: db.statement
            value: 'SELECT COUNT(*) FROM test_table WHERE str IS NOT NULL'
          -
            key: db.parameters
            value: '[]'
  -
    operationName: 'DBAL: TRANSACTION'
    tags:
      -
        key: db.transaction.end
        value: commit
    children:
      -
        operationName: 'DBAL: SELECT test_table'
        tags:
          -
            key: db.statement
            value: 'SELECT COUNT(*) FROM test_table WHERE str IS NOT NULL'
          -
            key: db.parameters
            value: '[]'
  -
    operationName: 'DBAL: UPDATE test_table'
    tags:
      -
        key: db.statement
        value: 'UPDATE test_table SET str = NULL WHERE str IS NOT NULL'
      -
        key: db.parameters
        value: '[]'
